src/core/src/package_managers/YumPackageManager.py

```diff
@@ -63,6 +63,14 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
                 and 'Critical' in installation_included_classifications and 'Security' in installation_included_classifications:
             self.composite_logger.log_debug("Updating classifications list to install all patches for the Auto Patching request since classification based patching is not available on CentOS machines")
             execution_config.included_classifications_list = [Constants.PackageClassification.CRITICAL, Constants.PackageClassification.SECURITY, Constants.PackageClassification.OTHER]
+        # Known errors and the corresponding action items
+        self.known_errors_and_fixes = {"SSL peer rejected your certificate as expired": self.fix_ssl_certificate_issue,
+                                       "Error: Cannot retrieve repository metadata (repomd.xml) for repository": self.fix_ssl_certificate_issue,
+                                       "Error: Failed to download metadata for repo":  self.fix_ssl_certificate_issue}
+        
+        self.yum_update_client_package = "sudo yum update -y --disablerepo='*' --enablerepo='*microsoft*'"
```

Comment:
This is from a different PR. Something's wrong if this is showing up in a diff here. @rane-rajasi, please look closer. @nikhim-um, please sanity check merge and status of master.

Reply:
Also blank lines below this and none above this make it look more odd.

Reply:
Yes, this was merged to master sometime this week. Could possibly be because this branch is fetching latest from your Auto Assessment PR branch? @nikhim-um since Koshy's changes are also merged in master, try fetching latest from master, maybe that will help